src/persist-types/src/codec_impls.rs

```diff
@@ -136,6 +140,9 @@ pub trait SimpleColumnarData {
     /// Type of [`arrow`] array the we decode data from.
     type ArrowColumn: arrow::array::Array + Clone + 'static;
 
+    /// The number of actual data bytes this item represents.
+    fn goodput(builder: &Self::ArrowBuilder) -> usize;
```

Comment:
I think of "goodput" as a bytes/s throughput measurement. Can we call this "goodbytes" instead? There's already some precedent for the term in persist

(Here and elsewhere)

Reply:
Yeah, apologies - Persist is very clear about this naming and I'm not sure how I managed to skate past it yesterday. Fixed here and elsewhere.